src/builder/Operations.cpp

```diff
@@ -475,6 +475,22 @@ void Nuanceur::Store(const CArrayUintValue& buffer, const CIntValue& index, cons
 	);
 }
 
+void Nuanceur::Store(const CArrayUint8Value& buffer, const CIntValue& index, const CUint16Value& value)
```

Comment:
This goes with the question in the PR in Play! Just wondering if there's anything preventing a uint16 to be stored in a uint16[]. If not, I think we should have a `CArrayUint16Value` (or maybe have that in a future PR).

Reply:
Saw your answer in the other repo. Even though this is not used, I think it should have a `CArrayUint16Value` parameter for future use.

Reply:
sure, once I get 16 bit store working, ill add do this as well.